Upstream-Status: Pending

jskwgen should be built by BUILD_CC in cross environment

7/23/2010 - created by Qing He <qing.he@intel.com>

diff -uNr js.orig//src/Makefile.ref js/src/Makefile.ref
--- js.orig//src/Makefile.ref	2009-02-25 21:24:29.000000000 +0800
+++ js/src/Makefile.ref	2010-07-23 13:26:51.000000000 +0800
@@ -303,10 +303,10 @@
 
 $(OBJDIR)/jskwgen.o: jskwgen.c jskeyword.tbl
 	@$(MAKE_OBJDIR)
-	$(CC) -o $@ -c $(CFLAGS) $<
+	$(BUILD_CC) -o $@ -c $(BUILD_CFLAGS) $<
 
 $(OBJDIR)/jskwgen$(HOST_BIN_SUFFIX): $(OBJDIR)/jskwgen.$(OBJ_SUFFIX)
-	$(CC) -o $@ $(CFLAGS) $(LDFLAGS) $^
+	$(BUILD_CC) -o $@ $(BUILD_CFLAGS) $(BUILD_LDFLAGS) $^
 
 endif
 
